README.md

```diff
@@ -6,69 +6,85 @@ Utilities for interacting with a TPM2.0 used for access to the Xaptum Edge Netwo
 
 [![Build Status](https://travis-ci.org/xaptum/xaptum-tpm.svg?branch=master)](https://travis-ci.org/xaptum/xaptum-tpm)
 
-## Requirements
-- cmake version >= 3.0
-- A C99-compliant compiler
-- Currently, only supports POSIX platforms
+## Installation
 
-## Building
+`xaptum-tpm` is available for the following distributions. It may also
+be built from source.
 
-`xaptum-tpm` uses CMake as its build system:
+### Debian Jessie
 
 ```bash
-# Create a subdirectory to hold the build
-mkdir -p build
-cd build
+# Add the repository to APT sources
+echo "deb http://dl.bintray.com/xaptum/deb jessie main" | sudo tee -a /etc/apt/sources.list
 
-# Configure the build
-cmake .. -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=ON
+# Install the repository GPG key
+apt-key adv --keyserver keyserver.ubuntu.com --recv-keys c615bfaa7fe1b4ca
 
```

Comment:
Should we explicitly include the `apt-get update` here, before the install?

Reply:
Yes indeed.